examples/java/src/main/java/org/apache/beam/examples/multilang/PythonDataframeWordCount.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can we add this to "examples/multi-language" instead ?
https://github.com/apache/beam/tree/master/examples/multi-language

Reply:
Moved.